### PR TITLE
fix(public-page-layout): update logos used in MC authentication flows

### PR DESCRIPTION
**Pull request by tylermorrisford**
#### Summary

This PR updates all logos within `packages/assets/logos/` to use the new branding, using the existing names.

#### Description

| PublicPageLayout Before      | PublicPageLayout After |
| ----------- | ----------- |
| Screenshot 2024-01-25 at 11 25 16 AM |  Screenshot 2024-01-25 at 11 35 05 AM|
| Navbar Before      | Navbar After |
|Screenshot 2024-01-25 at 11 36 38 AM | Screenshot 2024-01-25 at 11 39 28 AM |

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
I was expecting us to change all the logos at once to not live with inconsistencies.

We have quite a few other logos [here](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/assets/logos).

@FilPob What do you think about it?

**Comment by lidiyacommercetools** [comment]
> I was expecting us to change all the logos at once to not live with inconsistencies.
> 
> We have quite a few other logos [here](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/assets/logos).
> 
> @FilPob What do you think about it?

The only reason we decided to switch this particular logo is because we were specifically asked to by marketing earlier today. However, if this is problematic for a future implementation, we can discuss.

**Comment by FilPob** [comment]
@Shecki  are there any plans about when to replace all the logos in the MC? I think ideally we would do that in one go as Carlos suggested
@CarlosCortizasCT if we replace the files, can you send us links to the pages where these logos are used for reviewing?

**Comment by CarlosCortizasCT** [comment]
> @Shecki are there any plans about when to replace all the logos in the MC? I think ideally we would do that in one go as Carlos suggested @CarlosCortizasCT if we replace the files, can you send us links to the pages where these logos are used for reviewing?

@FilPob I'm not sure about it because in this PR we only change the logo in the `PublicPageLayout` component, which is exported and might be used from any consumer application.

I took a look at the Merchant Center monorepo and I only found one usage (in the login page).

image

**Comment by lidiyacommercetools** [comment]
@FilPob and @CarlosCortizasCT , to circle back, is this update of the image problematic? It was requested by marketing but the goal is to make the rebranding as unproblematic for Shield as possible. Will updating the one image be an issue for your team down the road?

**Comment by FilPob** [comment]
@lidiyacommercetools am coming back with the update from Sven :) He agreed that we should update all logos in MC at once. (no need to wait for the new colors) I can provide the updated logos this week... Not sure though what does that mean for this PR. Wdyt @CarlosCortizasCT ? should we merge it and update the other logos in a separate one or what?

**Comment by CarlosCortizasCT** [comment]
> @lidiyacommercetools am coming back with the update from Sven :) He agreed that we should update all logos in MC at once. (no need to wait for the new colors) I can provide the updated logos this week... Not sure though what does that mean for this PR. Wdyt @CarlosCortizasCT ? should we merge it and update the other logos in a separate one or what?

I think it would be better if we update all logos in the same PR.

**Comment by lidiyacommercetools** [comment]
@CarlosCortizasCT , that's fine. I will let marketing know.

**Comment by lidiyacommercetools** [comment]
@CarlosCortizasCT , to confirm, we will cancel this PR and I will inform marketing. And Shield will update all of the logos in the near future. Correct?

**Comment by CarlosCortizasCT** [comment]
> @CarlosCortizasCT , to confirm, we will cancel this PR and I will inform marketing. And Shield will update all of the logos in the near future. Correct?

@lidiyacommercetools we can do whatever is more convenient.

From my point of view, we can keep this PR open and, when @FilPob  provides the rest of the logos, he will share a link to them and Tyler can update this PR with them.

Would that work for you?

**Comment by lidiyacommercetools** [comment]
@CarlosCortizasCT , sounds like a plan. We will put this PR on hold until the other logos are ready to go.

**Comment by tylermorrisford** [comment]
Thank you @CarlosCortizasCT and @FilPob! @lidiyacommercetools I added the on hold label here.

**Comment by FilPob** [comment]
Here we go. all the updated logos [HERE](https://drive.google.com/drive/folders/1I6xpkO7xeZP4DFf1-DWaAjxq4NbPtN-p?usp=sharing) :)

**Comment by tylermorrisford** [comment]
Ok! @FilPob @CarlosCortizasCT @lidiyacommercetools All logos updated. I'm happy to find where they are used to and take screenshots in the visual testing app but may need some guidance as I'm not familiar with where all of these logos are used. Regardless thank you all for your help!

**Review by CarlosCortizasCT — APPROVED** [review]
This looks good to me now 💯 

Thanks a lot to everyone involved for your patience 🙇

**Comment by CarlosCortizasCT** [comment]
> Ok! @FilPob @CarlosCortizasCT @lidiyacommercetools All logos updated. I'm happy to find where they are used to and take screenshots in the visual testing app but may need some guidance as I'm not familiar with where all of these logos are used. Regardless thank you all for your help!

Regarding the Visual Testing Application, the changes can already be seen in [this Percy check](https://percy.io/commercetools-GmbH/app-kit-components/builds/32293463?utm_campaign=commercetools-GmbH&utm_content=app-kit-components&utm_source=github_status_public).

If @FilPob needs anything else, I think we can search for usages of this logos in MC frontend applications.

**Comment by FilPob** [comment]
Yeah it would be great to see some screenshots if everything looks as expected

**Comment by tylermorrisford** [comment]
> Yeah it would be great to see some screenshots if everything looks as expected

@FilPob I updated the description here 👆  with what I've found so far; it looks like the `application loader` image is loaded from assets/images/ instead of assets/logos/ - maybe we should update that in this PR as well? @CarlosCortizasCT WDYT?

**Comment by emmenko** [comment]
Thanks for the collaboration!

Two things are still missing:
- The changeset
- We also have an embedded svg logo on the index html template

**Comment by CarlosCortizasCT** [comment]
> > Yeah it would be great to see some screenshots if everything looks as expected
> 
> @FilPob I updated the description here 👆 with what I've found so far; it looks like the `application loader` image is loaded from assets/images/ instead of assets/logos/ - maybe we should update that in this PR as well? @CarlosCortizasCT WDYT?
> 
> Other than these two, I'm struggling to find where the rest of the logos are used.

I think we need to also have new logos for the ones we have in the `images` folder and, after we have them, I agree it would maybe make sense to move the to the `logos` directory.
@FilPob you can find the three SVGs we would need to have updated in [this directory](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/assets/images):
* [ct-logo.svg](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/assets/images/ct-logo.svg)
* [ct-logo_bw.svg](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/assets/images/ct_logo_bw.svg)
* [ct-logo_bw_white.svg](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/assets/images/ct_logo_bw_white.svg)

Regarding where the logos are used, I don't know neither.
I took a look at the MC frontend and couldn't find any usage.

**Comment by FilPob** [comment]
Sure, here they are: 
ct_logo_bw_white
ct_logo_bw
ct-logo

**Comment by FilPob** [comment]
Sure, here they are: 
ct_logo_bw_white
ct_logo_bw
ct-logo

**Comment by FilPob** [comment]
ct_logo_bw_white
ct_logo_bw
ct-logo

**Comment by FilPob** [comment]
sure here are the additional ones: https://drive.google.com/drive/folders/1JADQZ1pPwkbv-0l4MsBLdDC52UZt2OqC?usp=sharing

**Comment by tylermorrisford** [comment]
> Thanks for the collaboration!
> 
> Two things are still missing:
> 
> * The changeset
> * We also have an embedded svg logo on the index html template (`packages/mc-html-template/html-docs/application.html`)

Thank you @emmenko! Do we know which file should replace the embedded image in  `application.html` at line 50?

**Comment by tdeekens** [comment]
Should be here. You need to hand embed it. It's inlined for quick loading. I think we changed it 5 years ago the last time 🤗. 

Let us known if you need a guiding hand on Monday. It surely needs a release of app-kit. 

https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/mc-html-template/html-docs/application.html

**Comment by tylermorrisford** [comment]
> Should be here. You need to hand embed it. It's inlined for quick loading. I think we changed it 5 years ago the last time 🤗.
> 
> Let us known if you need a guiding hand on Monday. It surely needs a release of app-kit.
> 
> https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/mc-html-template/html-docs/application.html

Beauty - thank you @tdeekens! Updated in [6bf1e8b](https://github.com/commercetools/merchant-center-application-kit/pull/3379/commits/6bf1e8bac06c0ed31c61c906f444f72510a01e41)

**Review by stephsprinkle — APPROVED** [review]

**Comment by tylermorrisford** [comment]
Is it possible to get a screenshot of the `application.html` change without deploying? As far as I can tell the static assets are loaded from their environments. This is what I see if I load the updated `application.html` locally:
Screenshot 2024-01-29 at 8 03 24 AM

**Comment by stephsprinkle** [comment]
@commercetools/shield-team-design-system @FilPob I believe this PR takes care of updating all the logos in app kit and is ready for your final review.

**Review by FilPob — APPROVED** [review]

File: packages/mc-html-template/html-docs/application.html
Comment (CarlosCortizasCT) [inline packages/mc-html-template/html-docs/application.html:17]: I'm wondering about all this style changes 🤔 

I believe we should only be applying the expected inline svg changes.

Comment (tylermorrisford) [reply packages/mc-html-template/html-docs/application.html:17]: This change was made by Prettier - should I ignore Prettier for now?

Comment (CarlosCortizasCT) [reply packages/mc-html-template/html-docs/application.html:17]: Yes, that would be great. Please 🙏

**Comment by CarlosCortizasCT** [comment]
Since @emmenko also [requested some changes](https://github.com/commercetools/merchant-center-application-kit/pull/3379#issuecomment-1911530303) (already applied) I would like to wait for his approval too.

**Review by emmenko — APPROVED** [review]
Thanks! 💯

**Pull request merged**
